src/components/Img.js

```diff
@@ -0,0 +1,40 @@
+import axios from "axios"
+import React, { useEffect, useState } from "react";
+import ImageElement from "./ImgEle.js"
+import TitleElement from "./TitleEle.js"
+import DateElement from "./DateEle.js"
+import ExplanationElement from "./ExplanationEle.js"
+function CardContent(){
+    const [nasa, setNasa] = useState([]);
+    useEffect(() => {
+    
+    
+    axios.get('https://api.nasa.gov/planetary/apod?api_key=DEMO_KEY')
+    .then(returning => {      
+      const photo = returning.data
+      setNasa(photo)
+      console.log(returning)
+      const title = returning.data
+      setNasa(title)
+      const date = returning.data
+      setNasa(date)
+      const explanation = returning.data
+      setNasa(explanation)
```

Comment:
You should only have to do this once, because each time you are essentially running `setNasa(returning.data)`. I assume this is because you were using individual state variables before and switched over, but this is your only commit so I can't verify that. Try to commit more frequently.